supervisor/core.py

```diff
@@ -194,6 +194,9 @@ async def start(self):
         """Start Supervisor orchestration."""
         self.state = CoreState.STARTUP
 
+        # Evaluate the system
+        await self.sys_resolution.evaluate.evaluate_system()
```

Comment:
Hm, is this additional evaluation really necessary? It seems to me that the one at the end of setup just above should be enough? Or maybe we can drop that one instead?